etc/config-template.toml

```diff
@@ -1295,7 +1296,7 @@
 ## When Backup region [a,e) size exceeds `sst-max-size`, it will be backuped into several Files [a,b),
 ## [b,c), [c,d), [d,e) and the size of [a,b), [b,c), [c,d) will be `sst-max-size` (or a
 ## little larger).
-# sst-max-size = "144MB"
```

Comment:
Maybe it should be changed to the default sst file size (64MB)?

Reply:
The default value of `sst_max_size` is equal to `region_max_size` by desgin.
https://github.com/tikv/tikv/blob/7c509085cbe59c82b51c853abf230cd7f188b5a0/src/config/mod.rs#L2898

Reply:
/cc @YuJuncen

Reply:
In most cases, the SST size should be equal to the region size. That is we want to align the SST's boundary with the regions' because we use the SST's boundary to split regions during restoring. 

This `sst_max_size` is used for handing abnormal sized regions -- we generate SSTs in memory, when there are huge SSTs, OOM may happen.  

I guess both increase it to 384MiB or keep it unchanged are acceptable.

Reply:
/cc @3pointer

Reply:
The region size is uncompressed data size, but the sst size is compressed data size.

Reply:
please remember ping lightning developers when region size is changed. Some E2E failed recently.

https://github.com/pingcap/tidb/issues/55374#issuecomment-2285286504

Reply:
IMO, it's better to mark this information as an relevant info to TiKV, as the developers of TiKV may not have received this information.